### PR TITLE
Ria 5583 notifications for submit edited application

### PR DESCRIPTION
**Pull request by Marcellofabbri**
### JIRA link (if applicable) ###
[RIA-5583](https://tools.hmcts.net/jira/browse/RIA-5583)

### Change description ###
- Added all personalization classes and configs for notifications to be sent when an edited application is submitted

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

WAITING FOR THE EVENT TO BE AVAILABLE (WHEN 5580 IS DONE)

**Added suppression for CVE-2022-22978**

### CODE REVIEW

File: src/main/resources/application.yaml [hunk 1]
```diff
@@ -982,6 +988,7 @@ security:
       - "uploadBailSummary"
       - "uploadDocuments"
       - "makeNewApplication"
+      - "editBailApplicationAfterSubmit"
```
Comment (ThomasKKC) [inline src/main/resources/application.yaml:991]: Same as above

File: src/main/resources/application.yaml [hunk 2]
```diff
@@ -1096,6 +1103,7 @@ nbcEmailAddress: ${IA_NBC_EMAIL}
 ctscEmailAddress: ${IA_CTSC_EMAIL}
 ctscAdminPendingPaymentEmailAddress: ${IA_CTSC_ADMIN_PENDING_PAYMENT_EMAIL:pendingPayment@example.com}
 bailHomeOfficeEmailAddress: ${IA_BAIL_HOME_OFFICE_EMAIL:ho-bail-alar@example.com}
+bailHomeOfficeEmailAddressForApplicationSubmittedEdited: ${IA_BAIL_HOME_OFFICE_EMAIL_APP_EDITED_SUBMITTED:ALARAppealsBailTeam@homeoffice.gov.uk}
```
Comment (ThomasKKC) [inline src/main/resources/application.yaml:1106]: Does this not mean if we test in DEV/AAT/DEMO we're going to send to the real email? we should only have dummy emails here. 
Then add the real email in secret vaults in different ENV via bootstrap, Jenkinsfile_CNP, Jenkinsfile_nightly, Jenkinsfile_para

Comment (ThomasKKC) [reply src/main/resources/application.yaml:1106]: ${IA_BAIL_HOME_OFFICE_EMAIL:[ALARAppealsBailTeam@homeoffice.gov.uk]

Comment (Marcellofabbri) [reply src/main/resources/application.yaml]: I've deleted that line and am using `bailHomeOfficeEmailAddress` like in all other HO personalizations

File: src/main/resources/application.yaml [hunk 3]
```diff
@@ -883,6 +887,7 @@ security:
       - "submitApplication"
       - "uploadDocuments"
       - "makeNewApplication"
+      - "editBailApplicationAfterSubmit"
```
[the remark this answers is not in the document]
Comment (Marcellofabbri) [reply src/main/resources/application.yaml]: Looks like it, I've deleted all permissions for this event except for Admin

**Review by ThomasKKC — APPROVED** [review]

**Pull request merged**